### PR TITLE
Add Noble config

### PR DESCRIPTION
**Pull request by 0xh3rman**
Also added WalletConnect namespace for Cosmos

### CODE REVIEW

File: gemstone/src/wallet_connect/mod.rs [hunk 1]
```diff
@@ -51,20 +56,25 @@ pub fn get_reference(chain: Chain) -> Option<String> {
...
         | Chain::Blast => Some(chain.network_id().to_string()),
         Chain::Solana => Some("4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ".to_string()),
+        Chain::Cosmos
+        | Chain::Osmosis
+        | Chain::Celestia
+        | Chain::Noble
+        | Chain::Sei
+        | Chain::Injective => Some(format!(
+            "{}/slip44:{}",
+            get_namespace(chain).unwrap(),
```
Comment (gemcoder21) [inline gemstone/src/wallet_connect/mod.rs:66]: let's avoid force unwrap here

**Review by gemcoder21 — COMMENTED** [review]
In order to use noble on the iOS side we need to support cosmos address conversion?

**Comment by 0xh3rman** [comment]
@gemcoder21 it's available on wallet core

File: gemstone/src/wallet_connect/mod.rs (hunk 1 above)
Comment (0xh3rman) [reply gemstone/src/wallet_connect/mod.rs]: Done

**Pull request merged**